cve_bin_tool/cli.py

```diff
@@ -12,34 +12,41 @@
 
 # Python 2 compatibility stuff
 from __future__ import print_function
-import sys
+
+import argparse
+import logging
+import multiprocessing
 import os
-import csv
 import platform
 import subprocess
-import logging
-import argparse
+import sys
 import textwrap
 import threading
-import pkg_resources
-import multiprocessing
 from collections import defaultdict
 
-from .version import VERSION
-from .util import DirWalk, inpath
-from .extractor import Extractor
-from .strings import Strings
-from .file import is_binary
-from .OutputEngine import OutputEngine
+import pkg_resources
 
+from .OutputEngine import OutputEngine
 from .cvedb import CVEDB, OLD_CACHE_DIR
+from .extractor import Extractor
+from .file import is_binary
 from .log import LOGGER
+from .strings import Strings
+from .util import DirWalk, inpath
+from .version import VERSION
 
 try:
     import queue
 except ImportError:
     import Queue as queue
 
+try:
+    from egg_updater import update_egg
```

Comment:
This is how you can import a specific file (prepending to sys.path and using importlib can sometimes result in the wrong file being imported if there's another file in the current working directory with the same name, although if you have a way around this I'd be stoked, just what I've observed).

Doing this ensures that if a developer has `cve-bin-tool` checked out via the Git repo and installed with `pip install -e` and they run the `cve-bin-tool` from a different directory than the root of the source tree (for example they are in `/mnt` scanning something) they'll still get `update_egg` run.

```python
import os
import importlib.util
from setuptools import setup

egg_updater_path = os.path.join(os.path.dirname(__file__), "..", "egg_updater.py")
if os.path.isfile(egg_updater_path):
    spec = importlib.util.spec_from_file_location("egg_updater", egg_updater_path)
    egg_updater = importlib.util.module_from_spec(spec)
    spec.loader.exec_module(egg_updater)
else:
    egg_updater = None
```

Reply:
I have applied this changes